app/src/main/java/net/gsantner/markor/format/general/CommonTextActions.java

```diff
@@ -108,11 +121,9 @@ public boolean runAction(String action) {
                     } else if (callbackPayload.equals(rstr(R.string.char_punctation_mark_arrows))) {
                         _hlEditor.insertOrReplaceTextOnCursor("»«");
                     } else if (callbackPayload.equals(rstr(R.string.indent))) {
-                        _hlEditor.simulateKeyPress(KeyEvent.KEYCODE_MOVE_HOME);
-                        indentCurrentLine();
+                        runIndentLines(false);
                     } else if (callbackPayload.equals(rstr(R.string.deindent))) {
-                        _hlEditor.simulateKeyPress(KeyEvent.KEYCODE_MOVE_HOME);
-                        deIndentCurrentLine();
+                        runIndentLines(true);
```

Comment:
I have not removed `indentCurrentLine` `deIndentCurrentLine`. It appears they are not used anywhere. @gsantner Would you like me to delete them?

Reply:
you can remove old functions